pages/index.js

```diff
@@ -77,6 +81,88 @@ export default function Home({
         <CardGrid>{categories}</CardGrid>
       </section>
 
+      {/* your situation section */}
+      <section id="eligibility_criteria" className="">
+        <h3 className="text-2xl text-bold py-3">{t("eligibilityCriteria")}</h3>
+        <CriteriaGrid>
+          {/* location picker */}
+          <CriteriaBox>
+            <SelectPicker
```

Comment:
Looks messy, but it's perfect for now, until we can figure out how we're going to handle this.